sql/core/src/test/resources/sql-tests/inputs/transform.sql

```diff
@@ -0,0 +1,72 @@
+-- Test data.
+CREATE OR REPLACE TEMPORARY VIEW t1 AS SELECT * FROM VALUES
+('1', true, unhex('537061726B2053514C'), tinyint(1), smallint(100), array_position(array(3, 2, 1), 1), float(1.0), 1.0, Decimal(1.0), timestamp('1997-01-02'), date('2000-04-01')),
+('2', false, unhex('537061726B2053514C'), tinyint(2), smallint(200), array_position(array(3, 2, 1), 2), float(2.0), 2.0, Decimal(2.0), timestamp('1997-01-02 03:04:05'), date('2000-04-02')),
+('3', true, unhex('537061726B2053514C'), tinyint(3), smallint(300), array_position(array(3, 2, 1), 1), float(3.0), 3.0, Decimal(3.0), timestamp('1997-02-10 17:32:01-08'), date('2000-04-03'))
```

Comment:
Where's `int`? And, we need `array_position(array(3, 2, 1), 1)` for tests? `array_position(array(3, 2, 1), 1)` -> `bigint(3)`?

Reply:
> Where's `int`? And, we need `array_position(array(3, 2, 1), 1)` for tests? `array_position(array(3, 2, 1), 1)` -> `bigint(3)`?

Done